tsconfig.common.json

```diff
@@ -8,15 +8,15 @@
   },
   "compilerOptions": {
     "module": "es6",
-    "target": "es5",
+    "target": "es6",
```

Comment:
Private field are only supported in Ecmascript 2015/es6 or above.
We want SDK v10 to target es6